### ISSUE TITLE
Add tabbing support to keyboard navigation menu

### ISSUE THREAD

**Issue by shicholas** [NeonLaw/codebase#832]
**Is your feature request related to a problem? Please describe.**
When a user opens the keyboard navigation menu with `Shift + /` they should be able to tab through the menu, and eventually hit the close modal button.

**Additional context**
You can see this for yourself at https://www.neonlaw.net
